commands/issue/issue.go

```diff
@@ -22,6 +23,19 @@ func NewCmdIssue(f *cmdutils.Factory) *cobra.Command {
 		Use:   "issue [command] [flags]",
 		Short: `Work with GitLab issues`,
 		Long:  ``,
+		Example: heredoc.Doc(`
+			$ glab issue list
+			$ glab issue create --label --confidential
+			$ glab issue view --web
+			$ glab issue note -m "closing because !123 was merged" <issue number>
+		`),
+		Annotations: map[string]string{
+			"help:arguments": heredoc.Doc(`
+				An issue can be supplied as argument in any of the following formats:
+				- by number, e.g. "123"; or
+				- by URL, e.g. "https://gitlab.com/OWNER/REPO/-/issues/123".
```

Comment:
I don't think `glab` supports specifying issues by URL yet. Do you think implementing that?

Reply:
I actually just assumed it did, I'll wait until somebody asks then I can look into implementing it